src/ORM/Table.php

```diff
@@ -243,6 +247,16 @@ public function __construct(array $config = [])
         if (!empty($config['associations'])) {
             $associations = $config['associations'];
         }
+        if (!empty($config['validator'])) {
+            if (!is_array($config['validator'])) {
+                $this->validator(null, $config['validator']);
```

Comment:
@ADmad it's resolved to "default" here: https://github.com/cakephp/cakephp/blob/3.0/src/ORM/Table.php#L1133

I agree with @deizel passing `null` to not unnecessarily expose the default value elsewhere in the code.